tests/mesh/mesh_smoother_test.C

```diff
@@ -101,12 +103,12 @@ private:
                     modulation *= (pj - 0.5) * (pj - 0.5) * 4.; // quadratic bump centered at 0.5
                   }
               }
-            output(i) = p(i) + (std::pow(xi, 3) - xi) * modulation;
+            const auto delta = (std::pow(xi, 3) - xi) * modulation;
```

Comment:
Please use `libMesh::Utility::pow<N>` for all integer powers. I know this isn't a new line of code in this PR, but I forgot to mention it on the earlier one. It is just a template function that compiles down to pure multiplication/no function calls at runtime.